soh/soh/Enhancements/randomizer/3drando/hint_list/hint_list_exclude_overworld.cpp

```diff
@@ -230,6 +230,14 @@ void HintTable_Init_Exclude_Overworld() {
                        Text{"#fishing in maturity# bestows", /*french*/"#pêcher dans sa maturité# promet", /*spanish*/"#pescar en la madurez# conduce a"},
   });
 
+    hintTable[LH_HYRULE_LOACH] = HintText::Exclude({
+                       //obscure text
+                       Text{"#fishing the legend# bestows", /*french*/"!!!", /*spanish*/"#pescar a la leyenda# conduce a"},
```

Comment:
Ask in the discord for Purple Hato to do a french translation if one of the other randomisers doesn't have one, they usually handle this stuff.